_posts/2014-04-02-using-modules-and-the-resolver.md

```diff
@@ -129,7 +131,7 @@ explicitly:
 
 {% highlight javascript linenos %}
 // app/helpers/trim.js
-export default function(value, options) {
+export default function([value], options) {
```

Comment:
we don't make babel mandatory yet, `[value]` should be made its ES5 equiv until babel becomes mandatory for JS users